src/main/java/net/runelite/client/plugins/microbot/banchecker/BanCheckerPlugin.java

```diff
@@ -0,0 +1,80 @@
+package net.runelite.client.plugins.microbot.banchecker;
+
+import com.google.inject.Provides;
+import net.runelite.api.events.GameStateChanged;
+import net.runelite.client.config.ConfigManager;
+import net.runelite.client.eventbus.Subscribe;
+import net.runelite.client.events.ConfigChanged;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+
+import javax.inject.Inject;
+@PluginDescriptor(
+        name = "Ban Checker",
+        description = "Checks if account is banned based on login screen.",
+        tags = {"ban", "checker"},
+        authors = {"BanaanBakje"},
+        version = BanCheckerPlugin.version,
+        iconUrl= "https://chsami.github.io/Microbot-Hub/BanCheckerPlugin/assets/icon.png",
+        isExternal = true,
+        enabledByDefault = false
```

Comment:
```suggestion
        enabledByDefault = PluginConstants.DEFAULT_ENABLED,
        isExternal = PluginConstants.IS_EXTERNAL
```
Just cleaner to use the PluginConstants for the default values